_topic_maps/_topic_map_rosa.yml

```diff
@@ -88,6 +88,13 @@ Topics:
 - Name: Deleting a ROSA cluster
   File: rosa-sts-deleting-cluster
 ---
+Name: Red Hat OpenShift Cluster Manager
+Dir: ocm
+Distros: openshift-rosa
+Topics:
+- Name: Red Hat OpenShift Cluster Manager
```

Comment:
I am wondering if we can structure it so that we don't have the repetitive headings in the navigation. Right now you click Red Hat OpenShift Cluster Manager to see Red Hat OpenShift Cluster Manager again.